webapp/pom.xml

```diff
@@ -100,7 +100,7 @@
       <artifactId>trellis-webdav</artifactId>
       <version>${trellis.version}</version>
     </dependency>
-    <dependency>
+<!--     <dependency>
```

Comment:
One small question: why comment this out? Do we expect to bring it back at some point?

Reply:
Oh, whoops. No, I left it while running tests, but I'll take it out.